### PR TITLE
Outgoing Contract Calls

### PR DESCRIPTION
**Pull request by mvid**

### CODE REVIEW

File: module/x/gravity/keeper/keeper.go [hunk 1]
```diff
@@ -513,3 +513,53 @@ func (k Keeper) setLastObservedSignerSetTx(ctx sdk.Context, signerSet types.Sign
...
+	params := k.GetParams(ctx)
+
+	newContractCallTx := &types.ContractCallTx{
+		InvalidationNonce: invalidationNonce,
+		InvalidationScope: invalidationScope,
+		Address:           k.getBridgeContractAddress(ctx),
+		Payload:           payload,
+		Timeout:           params.TargetEthTxTimeout,
+		Tokens:            tokens,
+		Fees:              fees,
```
Comment (jackzampolin) [inline module/x/gravity/keeper/keeper.go:529]: Big Q here is how to pay these fees cc @zmanian

**Review by jackzampolin — APPROVED** [review]

**Comment by jackzampolin** [comment]
Looks great so far @mvid, I will you do hook implementation in this PR too?

**Comment by mvid** [comment]
@jackzampolin we already have hooks merged in from a previous PR, and they are being called in the EventHandler. Or are you talking about something else?

**Comment by jackzampolin** [comment]
🤦‍♂️

File: module/x/gravity/keeper/keeper.go (hunk 1 above)
Comment (jtremback) [reply module/x/gravity/keeper/keeper.go:529]: This is the biggest Q

**Pull request merged**